.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 parameters:
   image_suffix:
     type: string
-    default: '-v7e4468f'
+    default: '-dev-895be1f'
```

Comment:
Should be changed to a non dev version now that the the-process PR is merged.

Reply:
updated